src/Symfony/Component/Notifier/Message/ChatMessage.php

```diff
@@ -32,7 +31,7 @@ public function __construct(string $subject, MessageOptionsInterface $options =
         $this->options = $options;
     }
 
-    public static function fromNotification(Notification $notification, Recipient $recipient, string $transport = null): self
+    public static function fromNotification(Notification $notification): self
```

Comment:
Maybe we need $recipient here in the future? But being forced to pass an useless argument is worse in practice for me.